app/game_manager.py

```diff
@@ -191,6 +192,14 @@ def start_game(self) -> None:
         if self.show_welcome_screen() == "q":
             print(self.term.clear + self.term.exit_fullscreen)
         else:
+            # Show config loader
+            config = ConfigLoader()
+            response = config.config_loader_screen(self.term)
```

Comment:
do this before showing welcome screen 
welcome screen will only be shown if everything if running smoothly

Reply:
I like the welcome screen as the start of the app, that's how it is done on game apps if I remember correctly.